.gitmodules

```diff
@@ -20,3 +20,9 @@
 [submodule "src/3rd_party/simple-websocket-server"]
 	path = src/3rd_party/simple-websocket-server
 	url = https://github.com/marian-nmt/Simple-WebSocket-Server
+[submodule "src/3rd_party/ruy"]
```

Comment:
For all other submodules, we have their frozen forks in marian-nmt. I think we can have forks for ruy and simd_utils as well. This prevents from breakable changes, but delays pulling recent fixes, which seems what we prefer.

Reply:
all changes we needed to these are already in upstream. until we find we need to make local changes, doesn't it make sense to keep upstream? they can't do breakable changes to us until we manually bump our submodule commit.

Reply:
Okay, then that's fine. Thanks.